### ISSUE TITLE
Disabling Tensor Core for Specific CUDA Stream

### ISSUE THREAD

**Issue by snima** [NVIDIA/cutlass#289]
I am trying to run GEMM in double presision (input and output is double ) without using of tensor core and just with CUDA core. For that porpose I am using this line to disable tensor core for the stream that dgemm is going to performe : 

> cublasSetMathMode(handle, CUBLAS_DEFAULT_MATH);

But after profiling I am seeing: 

    cutlass::kernel<cutlass_80_tensorop_d884gemm_64x32_16x4_nn_align1>

Which means that tensor core has benn enabled again. Do you have any idea to keep it disabled?

The GPU that I am using is A100 and cuda 11.0.

Aha! Link: https://nvaiinfa.aha.io/features/CUTLASS-24

**Comment by hwu36** [comment]
https://docs.nvidia.com/cuda/cublas/index.html#cublasmath_t says that 



> CUBLAS_DEFAULT_MATH | This is the  default and highest-performance mode that uses compute and intermediate  storage precisions with at least the                                              same number of mantissa and  exponent bits as requested. Tensor Cores will be used whenever  possible.
> -- | --

That said, I don't think you can disable tensor core when using double precision in cublas.  Tensor core and CUDA core provide the same precision and there is no point for cublas to use CUDA cores on Ampere.

**Comment by mnicely** [comment]
This has been answered here https://forums.developer.nvidia.com/t/disable-tensor-cores-in-cublas-functions-explicity/188795

TL;DR - Can't disable FP64 TCs
